src/apps/activities/domain/response_values.py

```diff
@@ -37,6 +39,22 @@
 from apps.shared.domain import PublicModel, validate_color, validate_image, validate_uuid
 
 
+class PhrasalTemplateFieldType(str, Enum):
+    SENTENCE = "sentence"
+    ITEM_RESPONSE = "item_response"
+    LINE_BREAK = "line_break"
+
+
+class PhrasalTemplateDisplayMode(str, Enum):
+    BULLET_LIST = "bullet_list"
+    SENTENCE = "sentence"
+    ITEM_INDEX = "item_index"
```

Comment:
I think the intention is to use this with the `ResponseType.SLIDERROWS` response type, but it doesn't tell me anything about how the response is to be displayed. Do you think it would make sense to remove this so that `ResponseType.SLIDERROWS` can be used with `PhrasalTemplateDisplayMode.SENTENCE` (like `ResponseType.SLIDER`)?

Reply:
done!